packages/material-ui/src/Snackbar/Snackbar.test.js

```diff
@@ -56,21 +55,21 @@ describe('<Snackbar />', () => {
 
     it('should support synchronous onExited callback', () => {
       const messageCount = 2;
-      let wrapper;
+      let view;
       const handleCloseSpy = spy();
       const handleClose = () => {
-        wrapper.setProps({ open: false });
+        view.setProps({ open: false });
         handleCloseSpy();
       };
       const handleExitedSpy = spy();
       const handleExited = () => {
         handleExitedSpy();
         if (handleExitedSpy.callCount < messageCount) {
-          wrapper.setProps({ open: true });
+          view.setProps({ open: true });
         }
       };
       const duration = 250;
-      wrapper = mount(
+      view = render(
```

Comment:
maybe var name `view` could create misunderstanding, kcd suggests to use something similar to 'utils'

Reply:
Ultimately we only want to destructure `setProps`. The rest should be handled via `import { screen } from 'test/utils/createClientRender'`.

If I remember correctly Kent's stance was that he doesn't really care as long as it's not `wrapper` from enzyme. With the slight tendency to use `view` which I suggested. But `screen` and `view` are jargon anyway. Depending on your context `view` could be the large unit and `screen` the smaller one.

Reply:
Yes when it's possibile is better to use `screen`.
You idea makes sense, I was wondering that `view` could be confused with `wrapper` or similar

Reply:
Ahh, was curious about the use of screen vs destructuring :)

Reply:
With `screen` you can access to every API without the need to declare every variables, the only exception is for the `container` instance. If you need it you have to take from `render` return.